src/app/builders/hooks/useGetBuilderRewardsSummary.tsx

```diff
@@ -7,9 +7,8 @@ import { usePricesContext } from '@/shared/context'
 import { useReadGauges } from '@/shared/hooks/contracts'
 import { useMemo } from 'react'
 import { TOKENS } from '@/lib/tokens'
-import { Address } from 'viem'
 
-export const useGetBuilderRewardsSummary = (backer?: Address, currency = 'USD') => {
+export const useGetBuilderRewardsSummary = (currency = 'USD') => {
```

Comment:
I know you didn't change it, but let's use the constant for 'USD'

```suggestion
export const useGetBuilderRewardsSummary = (currency = USD) => {
```